docs/production.md

```diff
@@ -73,7 +73,7 @@ This use case assumes that you are on a box that has SSH access to the target bo
 
 For a release version in production:
 
-    ansible-playbook -l <target-host> playbooks/katello_31.yml
+    ansible-playbook -l <target-host> playbooks/katello.yml
```

Comment:
We now use variables to determine the version. This means you have to set ```foreman_version``` to ```1.14``` and ```katello_version``` to ```3.3```. In the vagrant solution we get this mapping from ```config/versions.yaml``` but ansible-playbook can't read that so you have to specify this as ```-e foreman_version=1.14 -e katello_version=3.3```.